plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/handler/NewApplicationHandler.java

```diff
@@ -32,6 +32,10 @@
 	public Object execute(ExecutionEvent event) throws ExecutionException {
 		try {
 			WizardUtils.openWizardDialog(new NewApplicationWizard(), HandlerUtil.getActiveShell(event));
+
+			// This technically sends the wrong event for the wrong 'command' id,
+			// but the refresh resource handler only looks at the selection... 
+			new RefreshResourceHandler().execute(event);  
```

Comment:
This workaround will only refresh the selected resource, so this might help you when coding but I dont think that this is even a valid general purpose workaround. I would remove it.